src/Tasks/GenerateResource.cs

```diff
@@ -56,7 +56,7 @@ public sealed partial class GenerateResource : TaskExtension
 
 
 #region Fields
-        
+
```

Comment:
Would you mind reverting the whitespace changes in this file? I think your editor (correctly) respected our `.editorconfig` but we haven't actually fixed that across the tree yet . . .

Reply:
Oh sorry, I have my text editor set to always delete trailing whitespace. Is that something that I shouldn't have?

Reply:
I just fixed it, I'll push it in just a second.

Reply:
I have no problem with fixing trailing whitespace, but I'd like to see it separated from the contents of this commit so it's easier to understand changes in the future.

Reply:
wait nope, that is the issue, I think. I'll fix it after everything else in something like notepad or something.

Reply:
sure, I'll revert it and then push it as a new commit.

Reply:
should be fixed.